paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -968,6 +968,17 @@
     func : prod
   backward : prod_grad
 
+- op : push_sparse_v2
+  args : (Tensor[] ids, Tensor[] w, Tensor[] out_grad_in, int embeddingdim = 11, int tableid = 0, str accessorclass = "", str ctrlabelname = "", int paddingid = 0, bool scalesparsegrad = true, str[] inputnames = {}, bool is_distributed = true)
+  output : Tensor[](out_grad_out){out_grad_in.size()}
+  infer_meta :
+    func : UnchangedInferMeta
```

```suggestion
    func : UnchangedMultiInferMeta
```
这里我之前忽略了变量类型的问题，改成这样就可以通过编译了

---

收到，麻烦大佬了。